src/javascript/ContentEditor/validation/validate.js

```diff
@@ -70,6 +70,44 @@ const colorFieldValidation = (values, field) => {
     return undefined;
 };
 
+const rangeFieldValidation = (values, field) => {
+    const error = 'invalidRange';
+    const supportedTypes = ['LONG', 'DECIMAL', 'DOUBLE'];
+    if (!supportedTypes.includes(field.requiredType) || !values[field.name]) {
+        return;
+    }
+
+    const fieldValues = Array.isArray(values[field.name]) ? values[field.name] : [values[field.name]];
+    // If one value is invalid, error !
+    if (fieldValues.some(value => isNaN(value))) {
+        return error;
+    }
+
+    if (!field.valueConstraints || field.valueConstraints.length === 0) {
+        return;
+    }
+
+    return field.valueConstraints
+        .map(constraint => {
+            try {
+                const {lowerBoundary, disableLowerBoundary, upperBoundary, disableUpperBoundary} = extractRangeConstraints(constraint.value.string);
+
+                return fieldValues
+                    .every(fieldValue => {
+                        // Lower boundary Check
+                        const checkStr = `${fieldValue} >${disableLowerBoundary ? '' : '='} ${lowerBoundary} && ${fieldValue} <${disableUpperBoundary ? '' : '='} ${upperBoundary}`;
+                        // eslint-disable-next-line no-eval
+                        return eval(checkStr);
```

Comment:
I would prefer to avoid `eval` if possible, maybe something like:

```
const checkBoundary = fieldValue >= Number(lowerBoundary) && fieldValue <= Number(upperBoundary) && 
    (!disableLowerBoundary || fieldValue !== lowerBoundary) &&
    (!disableUpperBoundary || fieldValue !== upperBoundary);
```

or if eval is needed, then at least add a check to add check to make sure lowerBoundary and upperBoundary are numbers